ios/FluentUI/Popup Menu/PopupMenuItem.swift

```diff
@@ -59,7 +59,35 @@ open class PopupMenuItem: NSObject, PopupMenuTemplateItem {
         self.executionMode = executionMode
         self.onSelected = onSelected
         self.isAccessoryCheckmarkVisible = isAccessoryCheckmarkVisible
+
         super.init()
+
+        initRequiredColors()
+
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(themeDidChange),
+                                               name: .didChangeTheme,
+                                               object: nil)
+    }
+
+    @objc private func themeDidChange(_ notification: Notification) {
+        initRequiredColors()
+    }
+
+    private func initRequiredColors() {
+        guard let accessoryView = self.accessoryView else {
+            let view = UIView()
```

Comment:
How expensive is it to create and throw away this UIView? Is there any other way to get a view to get the current fluent theme?

Reply:
Another way (besides making the color variables functions) would be to pass in a view in the init. But we would either need to go through all the scripts initializing `PopupMenuItem` and add a view in the init or make it an optional view which would still require us to create a view if nil.

Reply:
not a view, at this point it should pass in a token. but the property is there for existing clients to override the colors. ultimately everyone should be using override tokens but that more on @huwilkes future work.

Reply:
basically this nsobject class doesn't really need to know about uiview.

Reply:
So should we just go and pass in directly the color tokens in all the init calls?

Reply:
Could we just make these colors optionals just like the selected colors? Then we could provide the default tokens if these colors aren't set by the client.